pages/1.13/security/oss/login/external-user-login/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: layout.pug
+navigationTitle:  External User Login
+title: External User Login
+excerpt: Logging in to DC/OS as an external user
+menuWeight: 10
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+# Login using the DC/OS CLI
+
+**Prerequisite:**
+- [DC/OS CLI](/1.13/cli/)
+
+1.  To log in to the DC/OS CLI, enter the [auth login](/1.13/cli/command-reference/dcos-auth/dcos-auth-login/) command:
+
+```bash
+dcos auth login --provider dcos-oidc-auth0
+If your browser didn't open, please go to the following link:
+
+    http://172.17.0.2/login?redirect_uri=urn:ietf:wg:oauth:2.0:oob
```

Comment:
Ahh, this is an example output. I think keeping the default IP address here for `minidcos` is ok.

Reply:
There's a whole RFC for IP addresses to use in documentation: https://tools.ietf.org/html/rfc5737

Although don't use 198.51.100.0/24 because we use that for Spartan (against the RFC):
```
These blocks are not for local use
```

Reply:
So funny :dagger: